frontend/src/cljs/imas_seamap/map/views.cljs

```diff
@@ -217,7 +218,7 @@
       ;; then orders in the map by update not by list):
        (map-indexed
         (fn [i {:keys [server_url layer_name style id] :as layer}]
-          ^{:key (str id)}
+          ^{:key (str id (boundary-filter layer))}
```

Comment:
Good job on finding this (not sure how long it would have taken me!), but I'm curious about it now.  It's effectively saying that layers are considered immutable by react-leaflet I think (which might be a bug?); they won't re-render just because those props have changed, you need to force it by giving it a new id.

(good lord RL has changed a bit!  It's all react hooks and contexts and providers now; I had a patch or two incorporated in the early versions, and it was fairly easy to understand how it translated react prop changes to the underlying leaflet element.  Now... I have no idea)

Reply:
Thanks 😅